R/transformCounts.R

```diff
@@ -98,6 +98,14 @@
 #' log2 = log2(x)}
 #' where \eqn{x} is a single value of data.
 #'
+#' \item 'difference': Pairwise differences between features.
```

Comment:
Just thinking if we should have shorter name forms (or aliases) like "diff" and "div"? Or perhaps they are too unclear(?). Comment from @TuomasBorman could be welcome.

Reply:
I would keep full names to avoid unclarity. We could add these aliases later if needed

Reply:
Could it be just "-" ... and for division "/" ... then how about supporting the same for sums (+) and products (*).. (no immediate need but would be easy to implement on the same go..